ts/Core/Tooltip.ts

```diff
@@ -1798,10 +1799,23 @@ class Tooltip {
 
             // Pad it by the border width and distance. Add 2 to make room for
             // the default shadow (#19314).
-            pad = (options.borderWidth || 0) + 2 * distance + 2;
+            pad = (
+                (options.borderWidth || 0) +
+                2 *
+                distance +
+                2
+            );
+
+            const baseWidth = width + pad;
 
             renderer.setSize(
-                width + pad,
+                this.outside ? clamp(
```

Comment:
Wait a minute, found another nit to pick 😬 . The `renderer` [only exists](https://github.com/highcharts/highcharts/blob/a09cb5c5fe2ebe820e6f32be953cb9a42c6a31fd/ts/Core/Tooltip.ts#L475-L483) when `this.outside` is true. Which means we can skip the ternary here and always use the clamp.

Reply:
Good catch!